test/unit/krane/template_sets_test.rb

```diff
@@ -32,6 +33,16 @@ def test_template_sets_with_invalid_suffix_is_invalid
     assert_equal(template_sets.validate, expected)
   end
 
+  def erb_is_not_considered_valid_when_render_erb_is_false
+    bad_filepath = File.join(fixture_path("for_unit_tests"), "partials_test.yaml.erb")
+    template_sets = template_sets_from_paths(bad_filepath)
+    expected = [
+      "File #{bad_filepath} does not have valid suffix (supported suffixes: " \
+        ".yml, .yaml, or secrets.ejson)",
+    ]
+    assert_equal(template_sets.validate, expected)
+  end
```

Comment:
Functions that don't start with `test_` are not actually run in the test suite. Furthermore, this test currently fails, since `render_erb` defaults to true, there are no errors when rendering this. 

Furthermore, when we do explicitly set `render_erb: false`, the expected error doesn't match actual:

```
Failure:
TemplateSetsTest#test_erb_is_not_considered_valid_when_render_erb_is_false [/Users/timothysmith/src/github.com/Shopify/kubernetes-deploy/test/unit/krane/template_sets_test.rb:44]
Minitest::Assertion: --- expected
+++ actual
@@ -1 +1 @@
-["Rendering erb during deploy is not supported. Try piping the output of `krane render` to `krane-deploy` with the --stdin flag"]
+["File /Users/timothysmith/src/github.com/Shopify/kubernetes-deploy/test/fixtures/for_unit_tests/partials_test.yaml.erb does not have valid suffix (supported suffixes: .yml, .yaml, or secrets.ejson)"]
```

Reply:
:facepalm: Will fix this as soon as I have a coffee.